src/plasoscaffolder/dal/explain_query_plan.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=no-member
+# pylint does not recognize connect and close as member
+"""Class for the explain query plan"""
+
+from plasoscaffolder.dal import base_sql_query_execution
+
+
+class ExplainQueryPlan(object):
+  """Class representing the explain query plan"""
+
+  def __init__(self,
+               sql_execution: base_sql_query_execution.BaseSQLQueryExecution):
+
+    """Initializes the explain query plan
+
+    Args:
+      sql_execution (base_sql_query_execution.BaseSQLQueryExecution): the
+        helper to execute a query
+    """
+    self._sql_execution = sql_execution
```

Comment:
call super before using self unless there is a reason to do so, then please add a comment